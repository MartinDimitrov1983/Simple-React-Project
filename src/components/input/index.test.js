import React from 'react'
import { render } from '@testing-library/react'

import Input from './index'

test('Snapshot Input', () => {
    const { asFragment } = render(<Input />)
    expect(asFragment()).toMatchSnapshot()
})
