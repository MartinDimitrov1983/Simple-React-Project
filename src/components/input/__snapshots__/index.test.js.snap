// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Input 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <label
      class="label"
    >
      :
    </label>
    <input
      class="formControl"
      required=""
      value=""
    />
  </div>
</DocumentFragment>
`;
